src/features/smartSearch/components/SmartSearchDialog/QueryOverview/index.tsx

```diff
@@ -136,11 +139,30 @@ const QueryOverview = ({
       {hasSaveCancelButtons && (
         <DialogActions>
           <Box
+            alignItems="center"
             display="flex"
             justifyContent="flex-end"
             m={1}
             style={{ gap: '1rem' }}
           >
+            <Typography color={theme.palette.text.secondary}>
```

Comment:
I think we want to display this bit of text even if it does not have the submit/cancel-buttons? (aka when it is inside of making a view column smart search).

Reply:
Making this work is more complicated than what one might think. But I added the new "output" popper which I believe fulfils this purpose.